README.md

```diff
@@ -227,15 +234,20 @@ provides an opportunity for us to experiment with new design principles.
 
 ### How do I use this on Databricks?
 
-Koalas requires Databricks Runtime 5.x or above. For the regular Databricks Runtime, you should upgrade NumPy and pandas to the appropriate versions and then install Koalas. This can be done using the Libraries tab on the cluster UI, or using dbutils in a notebook:
+Koalas requires Databricks Runtime 5.x or above. For the regular Databricks Runtime, you can install Koalas using the Libraries tab on the cluster UI, or using dbutils in a notebook:
 
 ```python
-dbutils.library.installPyPI("numpy", "1.16.2")
-dbutils.library.installPyPI("pandas", "0.24.2")
 dbutils.library.installPyPI("koalas")
 dbutils.library.restartPython()
 ```
 
+If you are trying `Koalas<0.4`, you should upgrade NumPy and pandas to the appropriate versions prior to installing Koalas:
```

Comment:
do we need this? that's a really old version.

Reply:
sure, let me remove it.